android/app/src/main/java/org/fossasia/openevent/viewmodels/UserProfileActivityViewModel.java

```diff
@@ -0,0 +1,73 @@
+package org.fossasia.openevent.viewmodels;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+import org.fossasia.openevent.api.APIClient;
+import org.fossasia.openevent.data.auth.User;
+import org.fossasia.openevent.dbutils.LiveRealmDataObject;
+import org.fossasia.openevent.dbutils.RealmDataRepository;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+import timber.log.Timber;
+
+public class UserProfileActivityViewModel extends ViewModel {
+    private CompositeDisposable compositeDisposable;
```

Comment:
Make it final

Reply:
why should it be final? we need to addDisposables to this variable.

Reply:
Making it final won't stop you from adding disposables to it. Disposables are it's fields while making it final would stop you from creating a new instance of it.

Reply:
okay I will do it . thanks :)